src/poc/miner_poc_mgr.erl

```diff
@@ -336,8 +336,10 @@ handle_add_block_event(POCChallengeType, BlockHash, Ledger, State) when POCChall
             %% take care of GC
             ok = purge_local_pocs(Block, Ledger, State),
 
-            %% GC local pocs keys every 50 blocks
-            case BlockHeight rem 50 == 0 of
+            %% GC local pocs keys every 101 blocks
+            %% this is tied to the GC cadence window
+            %% for the key propopsals in ledger v1
+            case BlockHeight rem 101 == 0 of
```

Comment:
is there a way to tie it to a constant in ledger v1 then?

Reply:
we could make this an exported ledger function returning a static number if we wanted.

Reply:
maybe rip this block out into an exported function on the ledger and call it in both places? https://github.com/helium/blockchain-core/blob/d78c0abe8beaea8a487ccbb9ebe5646f8365a88f/src/ledger/v1/blockchain_ledger_v1.erl#L2366

Reply:
> maybe rip this block out into an exported function on the ledger and call it in both places? https://github.com/helium/blockchain-core/blob/d78c0abe8beaea8a487ccbb9ebe5646f8365a88f/src/ledger/v1/blockchain_ledger_v1.erl#L2366

worth putting on the list of cleanup tasks. I'll get this merged in the meantime